0-common/igluctl/src/main/scala/com.snowplowanalytics.iglu/ctl/File.scala

```diff
@@ -167,7 +167,14 @@ object File {
               val gapErrors = Common.checkSchemasConsistency(files.map(_.content.self)).leftMap(_.map(error => Error.ConsistencyError(error)))
               Ior.fromEither(gapErrors).putRight(files)
           }
-          checked.leftMap(_.concat(jsonSchemas.collect { case Left(e) => e }))
+
+          val errors = NonEmptyList.fromList(jsonSchemas.collect { case Left(e) => e })
```

Comment:
I think also maybe we could extract this whole chunk of code into a separate (pure?) funciton and write a test for it as too many things are going on here and we already had a bug.